docs/core-concepts/core-sdk/sdk-overview.md

```diff
@@ -0,0 +1,35 @@
+# .NET Core Software Development Kit (SDK) Overview 
+
+## Introduction
+.NET Core Software Development Kit is a set of libraries and tools that allow developers to create .NET Core applications
+and libraries. 
+
+It contains the following components:
+
+1. The .NET Core Command Line Tools that are used to build applications
+2. .NET Core (libraries and runtime) that allow applications to both be built and ran
+3. The `dotnet` driver for running the CLI commands as well as running applications
+
+This is the package that developers will most likely acquire. 
+
+## Acquiring the .NET Core SDK
+As with any tooling, the first thing is to get the tools to your machine. Depending on your scenario, you can either 
+use the native installers to install the SDK or use the installation shell script.
+
+The native installers are primarily meant for developer's machines. The SDK is distributed using each supported platform's 
+native install mechanism, for instance DEB packages on Ubuntu or MSI bundles on Windows. These installers will install 
+and set up the environment as needed for the user to use the SDK immediately after the install. However, they also 
+require administrative privileges on the machine. You can view the installation instructions on the
+[getting started page](https://aka.ms/dotnetcoregs).
```

Comment:
all feedback I left for overview.md applies to this topic as well since it seems to be the same text